x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_bulk_body.ts

```diff
@@ -71,3 +72,38 @@ export const buildBulkBody = ({
   };
   return signalHit;
 };
+
+export const buildSignalFromSequence = (
+  events: BaseSignalHit[],
+  ruleSO: SavedObject<RuleAlertAttributes>
+): SignalHit => {
+  const rule = buildRuleWithoutOverrides(ruleSO);
+  const signal: Signal = buildSignal(events, rule);
```

Comment:
nit, you don't need the `: Signal`
```suggestion
  const signal = buildSignal(events, rule);
```